src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```

Comment:
I see you don't have liners as you described in the PR summary. Hence in Microverse, we use best practice please input linter for reacting and destructure necessary parts of code, also do validations of elements.